### PR TITLE
add django-specific uuid and decimal scalars

### PR DESCRIPTION
**Pull request by maxmorlocke**
Provides scalars for commonly used django types - UUID and Decimal.  Incorrect decimal parsing can lead to unpleasant data quality issues, so providing an opinionated scalar that follow's django's best practices is advised.

Docs PR: https://github.com/mirumee/ariadne-website/pull/57

### CODE REVIEW

**Comment by kedlav** [comment]
This should be ready for review

File: tests/django/test_scalars.py [hunk 1]
```diff
@@ -74,6 +92,29 @@ def test_datetime_parser_raises_value_error_on_invalid_data():
...
+
+
+def test_uuid_parser_parses_uuid_string():
+    uuid_str = "bb7efd70-b1cd-11ea-a5af-0242ac130006"
+    assert parse_uuid_value(uuid_str) == uuid.UUID(uuid_str)
+
+
+def test_uuid_parser_raises_value_error_on_invalid_data():
+    with pytest.raises(ValueError):
+        parse_datetime_value("meow")
```
Comment (gdalmau) [inline tests/django/test_scalars.py:115]: Use `parse_uuid_value` instead

**Review by gdalmau — CHANGES_REQUESTED** [review]

**Review by gdalmau — APPROVED** [review]
Fix merge conflicts so the PR can be ready for review :)

**Comment by maxmorlocke** [comment]
Closing, present in ariadne_django, targeted for release 0.2.0

**Pull request closed without merging**